x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/flyout.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo, useState } from 'react';
+import { EuiFlyout, EuiFlyoutHeader, EuiTitle, EuiFlyoutBody } from '@elastic/eui';
+import { EuiSpacer, EuiTabs, EuiTab } from '@elastic/eui';
+import { MetadataTab } from './metadata/metadata';
+import type { HostNodeRow } from '../../hooks/use_hosts_table';
+import { useUnifiedSearchContext } from '../../hooks/use_unified_search';
+
+interface Props {
+  node: HostNodeRow;
+  closeFlyout: () => void;
+}
+
+const flyoutTabs = [MetadataTab];
+
+export const Flyout = ({ node, closeFlyout }: Props) => {
+  const { getDateRangeAsTimestamp } = useUnifiedSearchContext();
+
+  const { to, from } = getDateRangeAsTimestamp();
+
+  const currentTimeRange = useMemo(
+    () => ({
+      from,
+      interval: '1m',
+      to,
+    }),
+    [from, to]
+  );
```

Comment:
**nit:** this memoization won't have any benefit as `to` and `from` will change on each rerender, if you want a memorized value you'd prefer to invoke `getDateRangeAsTimestamp` inside the useMemo call:
```js
const currentTimeRange = useMemo(
  () => ({
    ...getDateRangeAsTimestamp(),
    interval: '1m'
  }),
  []
);
```

Reply:
Yeah make sense, moved 👍